src/lib/backend.ai-client-node.ts

```diff
@@ -1584,6 +1588,17 @@ class VFolder {
     return this.client._wrapWithPromise(rqst);
   }
 
+  /**
```

Comment:
This is just a question. I heard that now we only use the `backend.ai-client-esm.ts` without manual compilation of `backend.ai-client-node.ts`. However, I might be wrong, so my question is, do we still have to update and keep the traditional `backend.ai-client-node.ts`?